web/src/lib/utils/markdownReader.ts

```diff
@@ -209,7 +209,8 @@ type TaskGrayMatter = {
   requiresLocation?: boolean;
   industryId?: string;
   licenseCertificationClassification?: string;
-  summaryDescriptionMd: string;
+  summaryDescriptionMd?: string;
+  bodyHeading?: string;
```

Comment:
[sand] Same comment. If this is required in the CMS, it should be required here.